From: David Zeuthen <davidz@redhat.com>
Date: Sun, 1 Apr 2007 08:48:35 +0000 (-0400)
Subject: give back the "button" capability to keyboards
X-Git-Url: http://gitweb.freedesktop.org/?p=hal.git;a=commitdiff;h=e0d3327b68fc043e6ac8a1b0bdf97249476a3968

give back the "button" capability to keyboards

Without this, gnome-power-manager will not react to buttons on some
keyboards.
---

--- a/fdi/policy/10osvendor/10-keyboard-policy.fdi
+++ b/fdi/policy/10osvendor/10-keyboard-policy.fdi
@@ -9,6 +9,9 @@
       </match>
       <match key="info.capabilities" contains="input.keyboard">
 	<append key="info.addons" type="strlist">hald-addon-keyboard</append>
+        <match key="info.capabilities" contains_not="button">
+	  <append key="info.capabilities" type="strlist">button</append>
+        </match>
       </match>
     </match>
   </device>
